index.html

```diff
@@ -1869,6 +1865,16 @@ <h5 class="modal-title-craft"></h5>
                                                         <input type="number" step="10" min="0" max="2047" />
                                                     </td>
                                                 </tr>
+                                                <tr>
+                                                    <td name="digitalIdleOffset">
+                                                        <label>D-Shot Offset (%)</label>
+                                                        <input type="number" step="0.01" min="0" max="1" />
+                                                    </td>
+                                                    <td name="dynamic_idle_min_rpm">
+                                                        <label>Dynamic Idle Min RPM (* 100)</label>
```

Comment:
Have you tested making the window small? Usually this long titles give problems and this is the reason why we have some of them in two lines. It seems that our css is not smart enough to do that by it's own.

Reply:
The example screenshot was taken with the minimum main window size that we are supporting (1336 * 893) - it seems to fit.